wiki/Building_101.md

```diff
@@ -13,7 +13,7 @@ You must first install it on your server.
 
 There are also some additional dependencies, which are usually covered by the `base-devel` package group of your linux distribution.
 
-ArchLinux:
+Arch Linux:
 ```
 sudo pacman -S repo base-devel git-lfs
```

Comment:
Ik I wrote this line

But a LOT more repos are needed iirc

Along with a symlink to ncursesw.so from libncurses.so.5